examples/fiber-local-automatic.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Revolt\EventLoop;
+use Revolt\EventLoop\FiberLocal;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+/**
+ * This logger uses {@see FiberLocal} to automatically log a transaction identifier bound to the current fiber.
+ *
+ * This might be used to log the current URL, authenticated user, or request identifier in an HTTP server.
+ */
+final class Logger
+{
+    private int $nextId = 1;
+    private FiberLocal $transactionId;
+
+    public function __construct()
+    {
+        $this->transactionId = new FiberLocal(fn () => $this->nextId++);
+    }
+
+    public function setTransactionId(int $transactionId): void
+    {
+        $this->transactionId->set($transactionId);
+    }
```

Comment:
This function isn't required in this example, correct?

Reply:
That's a leftover, indeed. Just copied the other example and adjusted it.

Reply:
Thought so, it makes that example a bit confusing. Can you remove it?

Reply:
Sure, wanted to wait until I'm at the correct computer, but did it in the web UI now.

Reply:
Ah sorry wasn't aware. This is also why I usually mention that if I'm not at a workstation :).

Reply:
Actually I'm at a workstation, just not my personal one. :smile: